build/BranchInfo.props

```diff
@@ -27,7 +27,7 @@
     <MajorVersion>1</MajorVersion>
     <MinorVersion>4</MinorVersion>
     <PatchVersion>0</PatchVersion>
-    <PreReleaseLabel>preview</PreReleaseLabel>
+    <PreReleaseLabel>preview1</PreReleaseLabel>
```

Comment:
I think it should be preview2. Going from “preview” to “preview1” doesn’t really make sense IMO.

Reply:
We also need to bump the 0.16 PreReleaseLabel below.